dipy/workflows/reconst.py

```diff
@@ -861,3 +862,118 @@ def get_fitted_tensor(self, data, mask, bval, bvec, b0_threshold=50):
         dkfit = dkmodel.fit(data, mask)
 
         return dkfit, gtab
+
+
+class ReconstIvimFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'ivim'
+
+    def run(self, input_files, bvalues_files, bvectors_files, split_b_D=400,
+            split_b_S0=200, save_metrics=[],
```

Comment:
It would be good if you can add `mask_files` and `b0_threshold=50` as input arguments

Reply:
done! :+1: